baggers.js

```diff
@@ -14998,17 +14998,101 @@ var data = {
+          abstract:
+            "Comment choisir la bonne priorisation ? Quels sont les outils du PO pour choisir les fonctionnalités ayant le plus de valeur, les utilisateurs à inviter, les tâches techniques à intégrer au backlog ? Je vous propose des outils clairs pour les PO de tout niveau",
+          tags: ['Agile', 'Scrum', 'Produit', 'Valeur', 'Priorisation', 'Product Owner', 'PO'],
+          lang: ['fr', 'en'],
+        },
         {
           title:
             'Dark Agile : Le Scrum Master "chronomètre" est un danger pour l\'équipe',
           abstract:
             "Pourquoi le Scrum Master ne peut pas être un dev, un PO ou tout autre rôle que celui de Scrum Master? À quoi ça sert vraiment un Scrum Master ? Juste à inculquer Scrum à l'équipe ? Et ensuite, que fait-il ? C'est une mission temporaire ?",
           tags: ['Agile', 'Scrum', 'Coaching', 'Dark Agile'],
-          lang: ['fr'],
+          lang: ['fr', 'en'],
+        },
+        {
+          title:
+            'Management : Une équipe agile demande une nouvelle approche',
+          abstract:
+            "Comment faire pour manager une équipe agile ? L'agilité vient en contradiction avec les méthodes traditionnelles, pour se rapprocher plus des valeurs de Deming. Comment, quand on est manager, doit-on se comporter lorsqu'on travaille avec une équipe agile ?",
+          tags: ['Agile', 'Scrum', 'Coaching', 'Management'],
+          lang: ['fr', 'en'],
+        },
+        {
+          title:
+            'Comment rater ses communautés de pratiques',
+          abstract:
+            "Une des clés de l'efficacité d'un modèle à l'échelle est la capacité des équipes à communiquer et à avancer dans le même sens. Quelles sont les erreurs à faire (et donc à éviter) pour ne pas construire des communautés de pratiques de façon pérenne ?",
+          tags: ['Agile', 'Scrum', 'Coaching', 'Management', 'Dépendance', 'Collaboration', 'Equipes'],
+          lang: ['fr', 'en'],
+        },
+        {
+          title:
+            'Example Mapping : Atelier',
+          abstract:
+            "L'example Mapping est un atelier permettant de facilité le découpage des US et d'en sortir le maximum de valeur, tout en limitant le gâchis en ne faisant pas des choses inutiles pour le client ou l'utilisateur",
+          tags: ['Agile', 'Scrum', 'Produit', 'Valeur', 'Priorisation', 'Product Owner', 'PO', 'Atelier'],
+          lang: ['fr', 'en'],
+        },
+        {
+          title:
+            'Story Mapping : Atelier',
+          abstract:
+            "Le meilleur moyen de comprendre le besoin du client c'est en étudiant son workflow et en en tirant des hypothèses sur son besoin",
+          tags: ['Agile', 'Scrum', 'Produit', 'Valeur', 'Priorisation', 'Product Owner', 'PO', 'Atelier'],
+          lang: ['fr', 'en'],
+        },
+        {
+          title:
+            'Architecture : Comment on concoit dans une équipe agile ? Quelle est la place de l'architect technique ?',
```

Comment:
Syntax error here :) The bagger.js won't load anymore.

Reply:
Thanks @kassovix. I'm fixing it right now.